client/write.go

```diff
@@ -48,3 +53,28 @@ func (c *Client) Write(key, secret string, force bool) error {
 	}
 	return nil
 }
+
+func (c *Client) WriteFile(key, file string, force bool) error {
+
+	fh, err := os.Open(file)
+	if err != nil {
+		return err
+	}
+
+	fdata := make([]byte, 8192)
```

Comment:
Is it the limit? :D

Reply:
probably this is not the best practice to do this that way. This is kind of guess because we don't know what will be the size of a stored file after gzip compression. The limit depends on the backend that we are using. The approach changed here: https://github.com/AirHelp/treasury/pull/88/commits/0c7355daba2016ddbe7c57ef9eb97c4e96cdc4d4